catalogue/webapp/test/integration/works.test.js

```diff
@@ -10,12 +30,21 @@ describe('/works', () => {
     );
   });
 
-  it('includes the search query in the title', async () => {
+  it('renders search results', async () => {
     await page.type('.search-query', 'test');
     await page.keyboard.press('Enter');
     await page.waitForNavigation();
+
     await expect(page.title()).resolves.toMatch(
       'test | Catalogue search | Wellcome Collection'
     );
+
+    const element = await page.$('.main h2');
```

Comment:
Assuming this isn't a thing as we don't have those classes?

Reply:
Nah we _do_ have a `main` class.

![Screenshot 2020-02-21 at 11 03 53](https://user-images.githubusercontent.com/1394592/75029421-ea8c1580-5499-11ea-9017-c06b18fc115f.png)

Reply:
Does this mean `renders search results` === `true`?